### DISCUSSION TITLE
Unable to install Thirdweb deploy

### DISCUSSION THREAD

**Discussion by QsAdedipe** [Cyfrin/foundry-full-course-cu#1542] [Q&A]
Pls I was trying to install thirdweb deploy on my visual code this is the error I am getting below:

![Screenshot 2024-03-28 090758](https://github.com/Cyfrin/foundry-full-course-f23/assets/97874921/49f8fd97-0cfe-4021-be02-08b5e2464064)
![Screenshot 2024-03-28 090758](https://github.com/Cyfrin/foundry-full-course-f23/assets/97874921/025c6273-e11d-4416-b59a-5ebb741e7ecc)

**Comment by EngrPips** [comment]
Hello @QsAdedipe ,  I actually never install or use it myself so i am not very sure what could be going wrong but then i think their documentation would be a great place to start figuring out what is wrong . here is the link https://portal.thirdweb.com/contracts/deploy/overview to their documentation

**Comment by Niferu** [comment]
Hi @QsAdedipe 

This error tells you that you are trying to install version that doesnt exist.

To fix the error "@thirdweb-dev/wallets@2.4.26 not matching version found" when running the command `thirdweb deploy`, you can try upgrading to the latest version of the SDK by using the CLI command `npx thirdweb@latest` install from within your code project. This will ensure that you have the latest compatible version of the SDK installed.
